Makefile

```diff
@@ -9,6 +9,7 @@ GOVET_IGNORE_RULES = \
 # List of executables needed for 'make generate'
 GENERATE_DEPENDENCIES = \
 	github.com/golang/mock/mockgen \
+	github.com/gogo/protobuf/protoc-gen-gogoslick \
```

Comment:
We should run this by legal - these aren't verbatim copies of one of the licenses they've okayed, and we shouldn't make judgments about the importance of the differences. Let's just get a quick once-over and have these licenses added to the approved list.